src/pages/en/tutorial/1-setup/1.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Prepare your development environment
+setup: |
+  import Badge from '~/components/Badge.astro';
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+  import PreCheck from '~/components/tutorial/PreCheck.astro';
+---
+
+<PreCheck>
+  - check the background programming knowledge you will need to get started building with Astro
+  - install any tools that you will use to build your Astro website
+  - create online accounts needed to publish your site to the web
+</PreCheck>
+
+## Background Knowledge 
+
+This tutorial assumes some basic familiarity with **HTML**, **Markdown**, **CSS**, and a little **JavaScript**. But, you will be able to learn everything you need to know by following the tutorial instructions.
+
+## Development Tools Needed
+
+:::tip[stackblitz]
+Want to complete this tutorial in an online code editor instead? Use the [StackBlitz setup instructions](/en/tutorial/stackblitz/) and then skip ahead to Unit 2!
+:::
+
+### Terminal
+
+You will use a **command line (terminal)** to create your Astro project and to run key commands to build, develop, and test your site.
+
+You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
+
+You will also need to have **Node.js** installed. To check to see whether you already have a compatible version (v.14.18.0, or v16.12.0 or higher) installed: 
+
+1. Launch your terminal and type the following into the command line:
+
+    ```sh
+    node -v
+    ```
+
+2. If necessary, follow the instructions to [install Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) via the command line.
+
+:::tip[we'll use...]
+While you are editing your project, this tutorial will only use the terminal built in to your code editor.
+:::
```

Comment:
I think we should cut this: you can use either a standalone terminal or the one built into your editor, and we haven't introduced the latter yet.